covid19-etl/etl/ctp.py

```diff
@@ -215,7 +284,7 @@ def parse_row(self, row):
             "positiveTestsViral": "positiveTestsViral",
             "negativeTestsViral": "negativeTestsViral",
             "positiveCasesViral": "positiveCasesViral",
-            "deaths": "deathConfirmed",
+            # "deaths": "deathConfirmed",
```

Comment:
Does `deaths` is no longer a required field for CTP dataset? Because if you comment this out, then it won't be populated in the sheepdog data.

Reply:
It will be populated from here https://github.com/uc-cdis/covid19-tools/blob/d2021225d555d57e3de65cfd6754895c0e348a79/covid19-etl/etl/ctp.py#L327

Reply:
And it will only take `deaths` values from race-based CSV?